Surface_mesh/include/CGAL/draw_surface_mesh.h

```diff
@@ -12,6 +12,9 @@
 #ifndef CGAL_DRAW_SURFACE_MESH_H
 #define CGAL_DRAW_SURFACE_MESH_H
 
+#include <CGAL/Polygon_mesh_processing/compute_normal.h>
+#include <CGAL/Dynamic_property_map.h>
```

Comment:
Both include are useless AFAIS

Reply:
See https://github.com/CGAL/cgal/pull/4876#discussion_r461551666.

Reply:
Oh, right. They are not needed in `CGAL/draw_surface_mesh.h`. My fault. That was an intermediate step of the refactoring.